### ISSUE TITLE
levit vs levit_c

### ISSUE THREAD

**Issue by aashusri** [facebookresearch/LeViT#5]
Could you please tell me what is difference between levit and levit_c? levit_c is bit slower using spped_test.py? what is its accuracy on the imagenet?

**Comment by btgraham** [comment]
levit uses torch.nn.Linear and the BNC memory format.
levit_c uses torch.nn.Conv2d and the BCHW memory format.
Relative speed may be platform/batch size dependent.
Test predictions differ due to machine precision issues (test accuracies may differ by ±0.1%).
